tests/foreman/ui/test_contentview.py

```diff
@@ -702,6 +702,108 @@ def test_negative_add_components_to_non_composite(self):
 
     @run_only_on('sat')
     @tier2
+    def test_negative_add_unpublished_cv_to_composite(self):
```

Comment:
Above are CLI tests, below is another testcase that checking adding both published and unpublished CVs. Please check #3914